scripts/prepare_sector_network.py

```diff
@@ -1235,6 +1235,9 @@ def add_generation(n, costs, existing_capacities=0, existing_efficiencies=None):
             ),
             efficiency2=costs.at[carrier, "CO2 intensity"],
             lifetime=costs.at[generator, "lifetime"],
+            p_min_pu=cf_conventional[generator]["p_min_pu"] if "p_min_pu" in cf_conventional.get(generator,{}).keys() else 0,
```

Comment:
We can simplify the expression

```suggestion
            p_min_pu=cf_conventional.get(generator,{}).get("p_min_pu", 0),
```